engine/src/flutter/impeller/typographer/backends/skia/typographer_context_skia.cc

```diff
@@ -452,7 +452,8 @@ TypographerContextSkia::CollectNewGlyphs(
       for (const auto& glyph_position : run.GetGlyphPositions()) {
         Point subpixel = TextFrame::ComputeSubpixelPosition(
             glyph_position, scaled_font.font.GetAxisAlignment(),
-            frame->GetOffset(), frame->GetScale());
+            frame->GetTransform() *
+                Matrix::MakeTranslation(frame->GetOffset()));
```

Comment:
This is annoying, the transform here doesn't include the offset, but when you go to render it in TextContents it has the offset included.